src/psyclone/transformations.py

```diff
@@ -2011,23 +2176,48 @@ def _validate(self, node, location, position):
 
         # Check new location conforms to any data dependencies
         # This also checks the location and position arguments
+        if not options:
+            options = {}
+        position = options.get("position", "before")
         if not node.is_valid_location(location, position=position):
             raise TransformationError(
                 "In the Move transformation apply method, data dependencies "
                 "forbid the move to the new location")
-        # pylint: enable=no-self-use
 
-    def apply(self, node, location, position="before"):
+    def apply(self, node, location, options=None):
         '''Move the node represented by :py:obj:`node` before location
         :py:obj:`location` (which is also a node) by default and after
         if the optional `position` argument is set to 'after'. An
-        exception is raised if the move is invalid.'''
+        exception is raised if the move is invalid.
```

Comment:
We can drop this last line about the exception given that it is documented in the :raises: section below.

Reply:
Yes.